src/main/java/org/kafkahq/repositories/ConsumerGroupRepository.java

```diff
@@ -52,9 +56,11 @@
     }
 
     public ConsumerGroup findByName(String clusterId, String name) throws ExecutionException, InterruptedException {
-        Optional<ConsumerGroup> topics = this.findByName(clusterId, Collections.singletonList(name)).stream().findFirst();
+        Optional<ConsumerGroup> consumerGroup = this.findByName(clusterId, Collections.singletonList(name)).stream().findFirst();
+
+        consumerGroup.ifPresent(cg -> cg.getUsers().addAll(userRepository.findByResourceType(clusterId, ResourceType.GROUP, name)));
```

Comment:
Seems to be tricky here, you append data to a normally immutable object.
I prefer to pass data on construct on `ConsumerGroup` class instead